shuup/front/templates/shuup/front/macros/general.jinja

```diff
@@ -257,6 +257,14 @@
             </div>
             <div class="collapse navbar-collapse" id="admin-tools-menu">
                 <ul class="nav navbar-nav navbar-right">
+                {% set page = shuup.general.get_admin_url(object) %}
+                    {% if page %}
+                        <li> 
+                            {% set page_url = page["url"] %}
+                            {% set page_name = page["name"] %}
```

Comment:
you don't need to use these two `set`.. just use `page.url` and `page.name` directly